From: Giovanni Mascellani <gmascellani@codeweavers.com>
Subject: [PATCH] winex11.drv: Add missing Vulkan extensions.
Message-Id: <20210210151734.3354128-1-gmascellani@codeweavers.com>
Date: Wed, 10 Feb 2021 16:17:34 +0100

They were flagged by validation layers as required:
VK_EXT_direct_mode_display depends on VK_KHR_display and transitively
on VK_KHR_surface. VK_KHR_external_fence_capabilities is required by
VK_STRUCTURE_TYPE_PHYSICAL_DEVICE_ID_PROPERTIES.

Signed-off-by: Giovanni Mascellani <gmascellani@codeweavers.com>
---
 dlls/winex11.drv/xrandr.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/dlls/winex11.drv/xrandr.c b/dlls/winex11.drv/xrandr.c
index 11111111111..11111111111 100644
--- a/dlls/winex11.drv/xrandr.c
+++ b/dlls/winex11.drv/xrandr.c
@@ -874,6 +874,7 @@ static BOOL get_gpu_properties_from_vulkan( struct gdi_gpu *gpu, const XRRProvid
         "VK_EXT_direct_mode_display",
         "VK_KHR_display",
         VK_KHR_SURFACE_EXTENSION_NAME,
+        "VK_KHR_external_fence_capabilities",
     };
     const struct vulkan_funcs *vulkan_funcs = get_vulkan_driver( WINE_VULKAN_DRIVER_VERSION );
     VkResult (*pvkGetRandROutputDisplayEXT)( VkPhysicalDevice, Display *, RROutput, VkDisplayKHR * );

-- 
2.36.1

